packages/core/src/cli/commands.ts

```diff
@@ -82,5 +84,21 @@ export function runCli(): void {
       }
     });
 
+  devCommand
+    .description(`dev format: 'mf' library`)
```

Comment:
What is the difference between `dev` and `build --watch`. This is MF specified, it's not likely to merge into the common command, consider moving it to the MF plugin?

Reply:
build --watch is also production mode and hmr runtime will not be injected into browser.
use dev command can make hmr work normally

Reply:
`dev` command only used in rslib, so this may be add into rslib repo

Reply:
> dev command only used in rslib, so this may be add into rslib repo

How to use with the `dev` command? Can it work without Storybook?

Reply:
i will write it into doc if we make sure this command can be added into rslib repo, now we use `rslib dev:mf` can start the command. We will not use this command without storybook in normal

Reply:
> We will not use this command without storybook in normal

1. If it's Storybook only, we could add it to https://github.com/rspack-contrib/storybook-rsbuild/blob/main/packages/addon-rslib/package.json. I think we only need to export `composeCreateRsbuildConfig` from Rslib core and finish all other things in the addon.
2. If not, such as use with Rspress ( cc @Timeless0911 ), we could keep `initDevMFRsbuild` inside Rslib core, the and the rest part to Rspress/Storybok. However, since `initDevMFRsbuild` is quite simple, I'm also fine Rspress/Storybook duplicate it themself.

Reply:
yeah, i will follow the sugguestion 1

Reply:
> `build --watch is also production mode`

Rslib use fixed `mode: production` yet. I wonder how hmr runtime injected into browser, by NODE_ENV?

Reply:
i will change mode to `development` and use `rsbuild.startDevServer ` to start dev mode to ensure hmr https://github.com/web-infra-dev/rslib/pull/349/files#diff-4114b8f08217be27efd35a3891857cfabc6eb75aee8ab39ba304e4d1b1095323R1251

Reply:
i'm trying to move this to rsbuild storybook addon

Reply:
Just to ensure: IMO, user need to use @module-federation/storybook-addon alongside storybook-addon-rslib?

Reply:
What can user do only with `@module-federation/storybook-addon`?

Reply:
app project can only use `@module-federation/storybook-addon`, lib can not

Reply:
It's a little confused for me now. We should demonstrates the relationships in the document.